Sources/ProtobufferBuilder/Model.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  File.swift
+//  
+//
+//  Created by Nityananda on 24.11.20.
+//
+
+import Runtime
+
+// MARK: - Message and Message.Property
+
+struct Message: Equatable, Hashable {
+    struct Property: Equatable, Hashable {
+        let isRepeated: Bool
```

Comment:
are you planning to add `optional` in the future?

Reply:
I do now 🙂